packages/discord.js/src/structures/User.js

```diff
@@ -305,12 +305,60 @@ class User extends Base {
     json.bannerURL = this.banner ? this.bannerURL() : this.banner;
     return json;
   }
-
-  // These are here only for documentation purposes - they are implemented by TextBasedChannel
-  /* eslint-disable no-empty-function */
-  send() {}
 }
 
+
+/**
+ * Sends a message to this user.
+ * @method send
+ * @memberof User
+ * @instance
+ * @param {string|MessagePayload|MessageCreateOptions} options The options to provide
+ * @returns {Promise<Message>}
+ * @example
+ * // Send a direct message
+ * user.send('hello!')
+ *   .then(message => console.log(`Sent message: ${message.content}`))
+ *   .catch(console.error);
+ * @example
+ * // Send a remote file
+ * user.send({
+ *   files: ['https://cdn.discordapp.com/icons/222078108977594368/6e1019b3179d71046e463a75915e7244.png?size=2048']
+ * })
+ *   .then(console.log)
+ *   .catch(console.error);
+ * @example
+ * // Send a local file
+ * user.send({
+ *   files: [{
+ *     attachment: 'entire/path/to/file.jpg',
+ *     name: 'file.jpg',
+ *     description: 'A description of the file'
+ *   }]
+ * })
+ *   .then(console.log)
+ *   .catch(console.error);
+ * @example
+ * // Send an embed with a local image inside
+ * user.send({
+ *   content: 'This is an embed',
+ *   embeds: [
+ *     {
+ *       thumbnail: {
+ *         url: 'attachment://file.jpg'
+ *       }
+ *     }
```

Comment:
As in original i put there *raw* object
but should we show `EmbedBuilder`? the same with attachment

Reply:
I think the raw is fine, since we're documenting `User#send()`. And not how to use an `EmbedBuilder`. Although on the other hand it might be better to point new users to a better direction with a comment

Reply:
Four examples is a lot of bloat to be filling the docs page with. Is it really necessary? Do we have four on the other send examples?

Reply:
The changes only differ in variables and comments. [`TextBasedChannel#send()`](https://github.com/discordjs/discord.js/blob/229ad077ff52d8706d68ed4d31983619a32eba45/packages/discord.js/src/structures/interfaces/TextBasedChannel.js#L115) also has four examples. It should be shortened to 1-2 imo.

Reply:
Why not generalise the existing documentation to account for both users and channels?

Reply:
> Why not generalise the existing documentation to account for both users and channels?

Is possible, but maybe misleading as not everybody thinks about a user like a channel